### ISSUE TITLE
embedding发生错误

### ISSUE THREAD

**Issue by luckfu** [gpustack/llama-box#13]
错误信息如下：
```bash
./llama-box -c 8192 -np 4--host 0.0.0.0 --port 8081 -m /data/models/bge-m3-GGUF/bge-m3-FP16.gguf --embedding -ngl 99 -ngld 99
/home/runner/work/llama-box/llama-box/llama.cpp/src/llama.cpp:17572:GGML_ASSERT( seq_id < n_tokens && "seq_id cannot be larger than n_tokens with pooling_type ==CLS or RANK") failed
```
然后服务夯死，无法再访问

**Comment by thxCode** [comment]
@luckfu hi, please paste the result from `./llama-box --version` and `./llama-box --system-info` here to distinguish the llama-box.

BTW, can you also provide the following information:
-  the whole log
- the steps for us to reproduce: like the way you are using embedding API.

**Comment by linyinli** [comment]
@luckfu It seems like you’re missing a space between `4` and `--host`.

**Comment by luckfu** [comment]
我大概找到了问题，出在 -np 4 这里，在llama.cpp 里找到的参考：
I think this means one of your requests has less than 4 tokens. It's technically a bug that we need to fix, but I thought it's not high priority since I figured not much point in embedding such small data. Can you confirm this is the cause?
取消这个-np 4 参数，或者每次在embedding时加几个无效空格之类

**Comment by thxCode** [comment]
https://github.com/ggerganov/llama.cpp/issues/6722 cc @Finenyaco

**Comment by thxCode** [comment]
fixed in v0.0.98
